package.json

```diff
@@ -121,6 +121,7 @@
     "redux-thunk": "1.0.0",
     "rtlcss": "2.0.5",
     "sanitize-html": "1.11.1",
+    "scrollbar-width": "3.1.1",
```

Comment:
So, when I installed this, I did install using `npm install --save scrollbar-width`.  Not sure why the shrinkwrap didn't update.  Will have to rebuild.